tests/test_summary.py

```diff
@@ -317,6 +423,22 @@ def not_covered():
         assert squeezed[6] == "1 file skipped due to complete coverage."
         assert self.last_command_status == 0
 
+        report = self.report_from_command(
```

Comment:
I'm torn about adding all these markdown outputs to the tests.  Is there a way we can test `_report_text` and `_report_markdown` more directly to reduce the overhead?